core/src/main/java/io/aiven/klaw/service/EnvsClustersTenantsControllerService.java

```diff
@@ -886,6 +896,7 @@ public ApiResponse deleteEnvironment(String envId, String envType) throws KlawEx
     }
 
     try {
+      removeEnvironmentMapping(envId, tenantId, envType);
```

Comment:
```suggestion
      removeKafkaEnvironmentMapping(envId, tenantId, envType);
```
if this makes more clear ?